mod_reforged/hooks/skills/effects/taunted_effect.nut

```diff
@@ -19,11 +21,50 @@
 			});
 		}
 
+		if (this.getMeleeDefenseModifier() != 0)
+		{
+			ret.push({
+				id = 13,
+				type = "text",
+				icon = "ui/icons/melee_defense.png",
+				text = ::MSU.Text.colorizeValue(this.getMeleeDefenseModifier()) + " Melee Defense"
+			});
+		}
+
+		if (this.getRangedDefenseModifier() != 0)
+		{
+			ret.push({
+				id = 14,
+				type = "text",
+				icon = "ui/icons/ranged_defense.png",
+				text = ::MSU.Text.colorizeValue(this.getRangedDefenseModifier()) + " Ranged Defense"
+			});
+		}
+
 		return ret;
 	}
 
-	q.getTauntedTarget <- function()	// New helper-function to get the taunter
+	q.onUpdate = @(__original) function( _properties )
+	{
+		__original(_properties);
+
+		_properties.MeleeDefense += this.getMeleeDefenseModifier();
+		_properties.RangedDefense += this.getRangedDefenseModifier();
+	}
+
+// New Functions
+	q.getTauntedTarget <- function()
```

Comment:
I'd say it should be `getTaunter` because `TauntedTarget` is this guy who has been taunted.